customer service test scenario

prem225 opened the pull request.

sagaraltimetrik commented on src/test/specs/customerService.specs.ts:
```diff
@@ -0,0 +1,41 @@
+      getTitles = await service.titleContainer;
+      disableHtml = await getTitles[i].getHTML(false);
+      removeTags = await disableHtml.trim();
+      //addLogger(await `${removeTags}`);
```
Please fix/remove this commented code.

sagaraltimetrik commented on src/test/specs/customerService.specs.ts:
```diff
@@ -0,0 +1,41 @@
+    await service.scrollToHelp();
+  });
+  it("should hover on all titles", async () => {
+    let hoverAll: any, getTitles: any, disableHtml: string, removeTags: string;
```
Instead of using "any", would you please appropriate data type/ return type.

sagaraltimetrik commented on src/test/specs/customerService.specs.ts:
```diff
@@ -0,0 +1,41 @@
+  it("should hover on all titles", async () => {
+    let hoverAll: any, getTitles: any, disableHtml: string, removeTags: string;
+    let sortedTitles = new Array();
+    let replaceNull: any;
```
Instead of using "any", would you please appropriate data type/ return type

gayatri-gawali commented on test/specs/customerService.spec.ts:
```diff
@@ -0,0 +1,39 @@
+  it("should hover on all titles", async () => {
+    let hoverAll: any, getTitles: any, disableHtml: string, removeTags: string;
+    let sortedTitles = new Array();
+    let replaceNull: string;
```
give proper name to 'replaceNull'

gayatri-gawali commented on test/specs/customerService.spec.ts:
```diff
@@ -0,0 +1,39 @@
+    for (let i = 0; i < getTopics.length; i++) {
+      hoverAll = await (await getTopics[i].$("a")).click();
+      getTitles = await service.titleContainer;
+      disableHtml = await getTitles[i].getHTML(false);
+      removeTags = await disableHtml.trim();
```
give proper name to 'disableHtml' and 'removeTags'

prem225 commented:
### Ticket Number / Feature:
 Updated Customer Service test case scenario

### Description:
- Added code that enters in customer service tab, which redirects to customer service page.
- After the page displayed, it scrolls down to 'Browser Help' and hover all the topics.
- And, fetch all the headings from each topics from description

### Test Execution Proof:
```
[0-0] STEP: Learn how to...
[0-0] STEP: Shipping Delivery
[0-0] STEP: Amazon Prime
[0-0] STEP: Payments Pricing
[0-0] STEP: Amazon Pay
[0-0] STEP: Returns Refunds
[0-0] STEP: Ordering
[0-0] STEP: Managing Your Account
[0-0] STEP: Popular Help Topics
[0-0] STEP: More Help
[0-0] STEP: Amazon Business
[0-0] STEP: Other Topics
```
### Files Included:
- customerService.spec.ts
- customerService.page.ts
